server/index.js

```diff
@@ -0,0 +1,7 @@
+///////////////////////////////////////////
```

Comment:
I'm not sure why index.js and httpHandler.js are separate files.

Reply:
Good practice of separating modules